[IMP] sales: update pricelist info (#10066)
odoo/documentation: content/applications/sales/sales/products_prices/prices/pricing.rst

in the :guilabel:`Company` field. If this field is left blank, the pricelist is automatically
applied to all companies in the database.

It should also be noted that, in Odoo 17 (and above), a chatter section can be found on pricelist

> forms, which enhances the ability to communicate about it.

Reviewer: ```suggestion
forms, which enhances the ability to communicate about them.
```

in the :guilabel:`Company` field. If this field is left blank, the pricelist is automatically
applied to all companies in the database.

> It should also be noted that, in Odoo 17 (and above), a chatter section can be found on pricelist

Reviewer 2: would also add this to the **Note** block since it's similar version-specific content, and so it becomes easy to edit/remove later

However, the :guilabel:`Time-based rules` tab and :guilabel:`Configuration` tab are always the same,
regardless of the chosen :guilabel:`Pricelists` setting.

> Price Rules tab (multiple prices per product)

Reviewer 2: Don't need the parent heading label here. 
Since it's an h4 it's implied that we're still under the Price Rules tab h3

```suggestion
Multiple prices per product
```

For more information, check out the following section: :ref:`Multiple prices per product
<sales/multiple-prices-per-product>`.

> Price Rules tab (advanced price rules)

Reviewer 2: same here
```suggestion
Advanced price rules
```

Check out the :ref:`Advanced price rules (discounts, formulas) <sales/advanced-price-rules>` section
for detailed steps on how to add advanced price rules to a pricelist.

Recurring Prices tab
--------------------

> Recurring prices are specifically used with :doc:`subscription products
> </applications/sales/subscriptions/products>`.

Reviewer 3: Update path to be relative please :)
```suggestion
Recurring prices are specifically used with :doc:`subscription products
<../../../subscriptions/products>`.
```

> .. seealso::
>    :doc:`/applications/sales/subscriptions`
> 

Reviewer 3: Seems unnecessary to have a seealso to the same Subscriptions doc at the top and bottom of this section. Suggesting to remove this one and use the seealso at the bottom of the section 
```suggestion
```

:guilabel:`Recurring Plan` column.

> Click :guilabel:`Add a price rule`, and after :guilabel:`Products`, and potential :guilabel:`Product
> Variants`, are selected, select the blank field in the :guilabel:`Recurring Plan` column to reveal a

Reviewer: ```suggestion
Once :guilabel:`Products` and/or :guilabel:`Product Variants`  are selected, click :guilabel:`Add a price rule`, and select the blank field in the :guilabel:`Recurring Plan` column to reveal a
```

drop-down menu of pre-designated recurrence periods (e.g. `Monthly`, `Quarterly`, `Weekly`, etc.).

New recurrence periods can also be directly created from this column. To do so, type in the name for
the new :guilabel:`Recurring Plan`, then select :guilabel:`Create` from the resulting drop-down menu
to create the time period, which can be edited later.

Or, select :guilabel:`Create and edit...` to reveal a :guilabel:`Create Recurring Plan` pop-up form.

.. image:: pricing/time-period-popup.png
   :align: center
   :alt: Custom time period pop-up form in Odoo Sales.

From this pop-up form, the new recurrence period can be directly configured, with specific

> :guilabel:`Details`, :guilabel:`Self-Service`, and :guilabel:`Pricing` options. Then, when the

Reviewer: ```suggestion
:guilabel:`Details`, :guilabel:`Self-Service`, and :guilabel:`Pricing` options. When the
```

   :align: center
   :alt: Custom time period pop-up form in Odoo Sales.

Lastly, add the desired price for this recurring price rule in the :guilabel:`Recurring Price`
column.

.. seealso::

>    :doc:`/applications/sales/subscriptions`

Reviewer 3: Update path to be relative please 
```suggestion
   :doc:`../../../subscriptions`
```

Rental rules tab
----------------

> Under the :guilabel:`Rental rules` tab, specific price rules can be configured for various rental

Reviewer: Should "rules" be capitalized?

Author: In this instance, no. That's because we are supposed to represent the look (and capitalization) of tabs, fields, etc. as they appear on the UI. And, in this case, the "rules" of "Rental rules" is **not** capitalized. But great attention to detail. 👍 

![Screenshot 2024-07-17 142405](https://github.com/user-attachments/assets/3832e731-5cd6-4365-95b7-5dd588e2524b)

--------------------

Recurring prices are specifically used with :doc:`subscription products
</applications/sales/subscriptions/products>`.

.. seealso::
   :doc:`/applications/sales/subscriptions`

Under the :guilabel:`Recurring Prices` tab, the same functionality of the :guilabel:`Price Rules`
tab is present. The only difference being that a recurring time period can be applied in the
:guilabel:`Recurring Plan` column.

Once :guilabel:`Products` and/or :guilabel:`Product Variants` are selected, click :guilabel:`Add a
price rule`, and select the blank field in the :guilabel:`Recurring Plan` column to reveal a
drop-down menu of pre-designated recurrence periods (e.g. `Monthly`, `Quarterly`, `Weekly`, etc.).

New recurrence periods can also be created from this column. To do so, type in the name for the new
:guilabel:`Recurring Plan`, then select :guilabel:`Create` from the resulting drop-down menu to
create the time period, which can be edited later.

Or, select :guilabel:`Create and edit...` to reveal a :guilabel:`Create Recurring Plan` pop-up form.

.. image:: pricing/time-period-popup.png
   :align: center
   :alt: Custom time period pop-up form in Odoo Sales.

> From this pop-up form, the new recurrence period can be configured, with specific

Reviewer 2: would move this to the same paragraph as line 156 since it's a part of that supplemental instruction, otherwise it comes off as a new instruction in the main flow (which it's not, it's an aside)

Under the :guilabel:`Website` section, there are a few options that can be configured. In the
:guilabel:`Website` field, this pricelist can be applied to a specific website, if working in a
multi-website environment. If left blank, the pricelist is applied to all websites in the database.

Tick the :guilabel:`Selectable` checkbox to have this pricelist as a selectable option for
customers to choose as they shop. If the :guilabel:`Selectable` box is left unticked, customers
**cannot** select this pricelist for themselves as they shop.

Lastly, there is the option to add an :guilabel:`E-commerce Promotional Code`. To add a code, type
in the desired promo code that, when entered during the checkout process, applies the pricelist to
the customer, even if the customer does not fall into the previously-specified criteria.

Then, in the :guilabel:`Discounts` section, there is a :guilabel:`Discount Policy` field with two
options to choose from: :guilabel:`Discount included in the price` or :guilabel:`Show public price &
discount to the customer`.

If :guilabel:`Discount included in the price` is selected, the price shown to the customer already

> accounts for the unique discount being applied. However, if the :guilabel:`Show public price &

Reviewer: ```suggestion
accounts for the unique discount being applied. However, if :guilabel:`Show public price &
```

   :align: center
   :alt: The pricelist field in a customer detail form in Odoo Sales.

.. note::
   When a customer is added to the database, the default pricelist is automatically applied to them.
   There is **no way** to have a blank *Pricelist* field on a contact form. Even if that field is

>    left blank, the default pricelist appears when that contact form is opened again.

Reviewer 2: TIL ^^
